styles/styles.css

```diff
@@ -189,7 +189,8 @@ body {
 }
 
 .see-project-button,
-.about-me-button {
+.about-me-button,
+.submit-button {
```

Comment:
Button interactions such as hover, active (when it's pressed), disabled and enable (default state) should be styled as shown in the Figma template.
Figma design:
![image](https://user-images.githubusercontent.com/83596135/160711885-0441398b-a853-4dc8-bc3d-e85454461b6a.png)

- [ ] Kindly, impelentent the style as per figma design for button interaction. :100: